core/eolearn/tests/test_core_tasks.py

```diff
@@ -481,9 +481,34 @@ def test_create_eopatch():
     assert np.array_equal(patch.data["bands"], data)
 
 
-def test_merge_eopatches(test_eopatch):
-    task = MergeEOPatchesTask(time_dependent_op="max", timeless_op="concatenate")
+def test_merge_eopatches() -> None:
+    dummy_timestamps = [datetime(2017, 1, 14, 10, 13, 46), datetime(2017, 2, 10, 10, 1, 32)]
 
-    del test_eopatch.data["REFERENCE_SCENES"]  # wrong time dimension
+    patch1 = EOPatch(
+        data={"bands": np.zeros((2, 4, 4, 3), dtype=np.float32)},
+        mask_timeless={"LULC": np.arange(0, 16).reshape(4, 4, 1)},
+        bbox=DUMMY_BBOX,
+        timestamps=dummy_timestamps,
+    )
+    patch2 = EOPatch(
+        data={"bands": np.ones((2, 4, 4, 3), dtype=np.float32)},
+        mask_timeless={"LULC": np.arange(16, 32).reshape(4, 4, 1)},
+        bbox=DUMMY_BBOX,
+        timestamps=dummy_timestamps,
+    )
 
-    task.execute(test_eopatch, test_eopatch, test_eopatch)
+    merged_patch = MergeEOPatchesTask(time_dependent_op="max", timeless_op="concatenate")(patch1, patch2)
+
+    expected_patch = EOPatch(
+        data={"bands": np.ones((2, 4, 4, 3), dtype=np.float32)},
+        mask_timeless={"LULC": np.arange(0, 32).reshape((16, 2), order="F").reshape(4, 4, 2)},
```

Comment:
it is a bit difficult to understand, but the only other option I see is `np.concatenate([np.arange(0,16).reshape(...), ...])` which also feels a bit weird. So let's just leave it.

Reply:
this solution would also demands that concatenation is done out side EOPatch init (i have tried it and like it even less)